qutebrowser/utils/log.py

```diff
@@ -526,6 +529,33 @@ def hide_qt_warning(pattern: str, logger: str = 'qt') -> typing.Iterator[None]:
         logger_obj.removeFilter(log_filter)
 
 
+def init_from_config(conf: 'configmodule.ConfigContainer') -> None:
+    """Initialize logging settings from the config.
+
+    init_log is called before the config module is initialized, so config-based
+    initialization cannot be performed there.
+
+    Args:
+        conf: The global ConfigContainer.
+              This is passed rather than accessed via the module to avoid a
+              cyclic import.
+    """
+    assert ram_handler is not None
```

Comment:
Looks like that assumption [isn't true after all](https://travis-ci.org/github/qutebrowser/qutebrowser/jobs/689086186#L381), as it crashes in tests when starting qutebrowser with `--loglines=0`. Could you take a look at that?

Reply:
Aha! mypy catches a bug :)